.github/workflows/lint.yml

```diff
@@ -36,12 +28,11 @@ jobs:
         with:
           PATTERNS: |
             **/**.go
-            **/go.mod
-            **/go.sum
+            go.mod
+            go.sum
```

Comment:
Should there be a match for `api/go.{mod,sum}` as well, or should this rule be modified again before merging #1672?

Reply:
That change should land on the `feature/proto-upgrade` branch.